processes-endpoint.md

```diff
@@ -0,0 +1,107 @@
+# Admin Processes Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+This endpoint allows adminstrators to list and manipulate script processes created by the [Scripts endpoint](scripts-endpoint.md).
+
+## Execution List
+**GET /api/admin/processes**
+
+This endpoint will return a list of all created processes. The JSON response document is as follows:
+
+```
+{
+  "page": {
+      	"size": 5,
+      	"totalElements": 14,
+      	"totalPages": 3,
+      	"number": 0
+  },
+  "sort" : {
+    "by" : "name",
+    "order" : "asc"
+  },
+  "_embedded" : {
+    "processes" [
+      {
+        "processId" : "000003f1-a850-49de-af03-997272d834c9",
+        "scriptName" : "import",
+        "startTime" : "2017-11-22T10:29:11Z",
+        "status" : "RUNNING"
+        "_links" : {
+          "self" : {
+            "href" : "/api/admin/processes/000003f1-a850-49de-af03-997272d834c9"
+          }
+        }
+      },
+      {
+        "processId" : "d3007ea8-1cc0-481d-914a-06720a200edf",
+        "scriptName" : "metadata-export",
+        "startTime" : "2017-11-20T10:29:11Z",
+        "status" : "FAILED"
+        "_links" : {
+          "self" : {
+            "href" : "/api/admin/processes/000003f1-a850-49de-af03-997272d834c9"
+          }
+        }
+      },
+      ...
+    ]
+  }
+```
+## Execution Details
+**GET /api/admin/processes/<:process-id>**
+
+This endpoint will return details on the requested process.
+
+```
+{
+  "processId" : "000003f1-a850-49de-af03-997272d834c9",
+  "scriptName" : "import",
+  "startTime" : "2017-11-22T10:29:11Z",
+  "status" : "RUNNING"
+  "_links" : {
+    "self" : {
+      "href" : "/api/admin/processes/000003f1-a850-49de-af03-997272d834c9"
```

Comment:
you need to have the output and file link here

Reply:
Ok, I'll add that